app/services/analytics_events.rb

```diff
@@ -1925,9 +1925,9 @@ def idv_doc_auth_verify_polling_wait_visited(**extra)
   # @option proofing_results [String,nil] context.stages.threatmetrix.exception If an exception was encountered making the request to the vendor, its message is provided here.
   # @option proofing_results [Boolean] context.stages.threatmetrix.timed_out Whether the request to the vendor timed out.
   # @option proofing_results [String] context.stages.threatmetrix.transaction_id Vendor-specific transaction ID for the request.
+  # @option proofing_results [String] context.stages.threatmetrix.session_id Session ID associated with the response.
+  # @option proofing_results [String] context.stages.threatmetrix.account_lex_id LexID associated with the response.
```

Comment:
Did you already run this by @rebecca-e-wright et al? I like the idea of pulling this up, but it feels like changing where it's stored will slightly complicate existing queries. Just want to make sure they're not caught off guard. (The perpetual problem with reorganizing log data...)

Reply:
Within this change we are still logging these values in the same place. I am duplicating them here to simplify changes when we _do_ move them.

I did DM @rebecca-e-wright to make sure her team knows these changes are coming. This change should be fine; changes that actually move them will require some extra work.

Reply:
Oh! I misunderstood what was happening. This is perfect; :shipit:!